plugins/contact-resources/src/components/CreateEmployee.svelte

```diff
@@ -66,6 +68,7 @@
         provider: channel.provider
       })
     }
+    dispatch('created', id)
```

Comment:
Better to provide 
```
export let onCreate: (() => Promise<void>)|undefined
```

Since in this case it will be possible to properly show creation progress on Create button.

Reply:
Done